runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsConfig.java

```diff
@@ -66,6 +66,17 @@ default boolean clue()
 
 	@ConfigItem(
 		position = 3,
+		keyName = "killcount",
+		name = "Killcount Command",
+		description = "Configures whether the Killcount command is enabled"
+	)
+	default boolean killcount()
```

Comment:
Fix indentation

Reply:
Might sound stupid but I'm missing the wrong identation here.  Looks fine for me in IntelliJ (https://i.imgur.com/hgWuVOW.png). Could you show an example of correct identation?

Reply:
It was my browser, oops. Looks correct